crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -195,4 +198,26 @@ def sanity_checks
       end
     end
   end
+
+  def upgrade
+    if request.post?
+      respond_to do |format|
+        format.html do
+          render file: "public/503_upgrade.html"
+        end
```

Comment:
yes, it is, but anyone who hasn't installed crowbar yet will likely not want to go to the barclamp page. so there it makes sense IMO. and in `crowbarctl` we have a similar warning.